cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -605,43 +615,35 @@ define(["jquery", "edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers", "common/j
             });
 
             describe("Subsection", function() {
-                var getDisplayNameWrapper, setEditModalValues, mockServerValuesJson,
-                    selectDisableSpecialExams, selectBasicSettings, selectAdvancedSettings,
-                    selectAccessSettings, selectTimedExam, selectProctoredExam, selectPracticeExam,
+                var getDisplayNameWrapper, setEditModalValues, setContentVisibility,  mockServerValuesJson,
+                    selectDisableSpecialExams, selectTimedExam, selectProctoredExam, selectPracticeExam,
                     selectPrerequisite, selectLastPrerequisiteSubsection, checkOptionFieldVisibility;
 
                 getDisplayNameWrapper = function() {
                     return getItemHeaders('subsection').find('.wrapper-xblock-field');
                 };
 
-                setEditModalValues = function (start_date, due_date, grading_type, is_locked) {
+                setEditModalValues = function (start_date, due_date, grading_type) {
                     $("#start_date").val(start_date);
                     $("#due_date").val(due_date);
                     $("#grading_type").val(grading_type);
-                    $("#staff_lock").prop('checked', is_locked);
-                };
-
-                selectDisableSpecialExams = function() {
-                    this.$("input.no_special_exam").prop('checked', true).trigger('change');
-                };
-
-                selectBasicSettings = function() {
-                   this.$(".modal-section .settings-tab-button[data-tab='basic']").click();
                 };
 
-                selectAdvancedSettings = function() {
-                   this.$(".modal-section .settings-tab-button[data-tab='advanced']").click();
+                setContentVisibility = function (visibility) {
+                    if (visibility) {
```

Comment:
Is this defensive line of code really needed?  Sometimes being overly defensive hides real issues.